open-grant-proposals/filecoin-miner-map.md

```diff
@@ -0,0 +1,188 @@
+-->
+
+As shown in the previous section, summing up to 150,000 USD.
+
+## Maintenance and Upgrade Plans
+
+<!--
+Specify your team's long-term plans to maintain this software and upgrade it over time.
+-->
+
+We are excited with the Filecoin ecosystem and want to start this project as the first step to join the community. Maintenance and upgrade plans are to be determined after the project is reviewed.
+
+# Team
+
+## Team Members
+
+- Gavin Yeung (blockchain entrepreneur)
+- Charles Zhang (CTO)
+- Yanhao Zhang (Blockchain engineer)
+- Terence Ma (Blockchain and math)
+- Sofia Lee (UI & UE)
+
+## Team Member LinkedIn Profiles
+
+- [Gavin Yeung](https://www.linkedin.com/in/gavin-yeung-3a32141/)
+- [Charles Zhang](https://www.linkedin.com/in/charles4zzhang/)
+- [Yanhao Zhang](https://www.linkedin.com/in/%E8%A8%80%E8%B1%AA-%E5%BC%A0-5ba646169/)
+- [Terence Ma](https://www.linkedin.com/in/terencema1023/)
+- [Sofia Lee](https://urilee.hashbase.io/) (personal site)
+
+## Team Website
+
+<!--
+Please link to your team's website here (make sure it's `https`)
+-->
+
+- [www.koinbros.com](http://www.koinbros.com)
+- [www.cryptomover.com](https://www.cryptomover.com/)
+
+## Relevant Experience
+
+<!--
+Please describe (in words) your team's relevant experience, and why you think you are the right team to build this project. You can cite your team's prior experience in similar domains, doing similar dev work, individual team members' backgrounds, etc.
+-->
+
+Our founder Gavin Yeung is a seasoned entrepreneur.  Prior to founding Koinbros, he worked in a bulge bracket bank for a number of years.  Gavin has an MBA from the University of Chicago and Computer Science Master and Bachelor degrees from the University of California, Los Angeles.
+
+Our CTO, Dr. Charles Zhang, was a Computer Science Professor in the Chinese Academy of Sciences.  He was CTO in two prior startups in Beijing and has experience in managing high achieving engineering teams.  Charles has a Ph.D. in Computer Science from Hong Kong University and Master and Bachelor degrees from Tsinghua University in Beijing, China.
+
+
+Our software engineer, Yanhao Zhang, is a graduate of the Polytechnic University of Hong Kong, studying Master of Science in Information technology.
+
+Our Business Operations Manager, Terence Ma, is a graduate of the London School of Economics, studying Bachelor of Science in Maths and Economics.
+
+Our advisors include a tenure financial statistics Professor from Hong Kong University of Science and Technology (Dr. Ying Ying Li, University of Chicago, Statistics Ph.D. 2008), a hedge fund manager (Richard Warfield, HBS, EECS Berkeley) and a successful entrepreneur (Gigi Wang, Stanford).
+
+## Team code repositories
+
+<!--
+Please provide links to your team's prior code repos for similar or related projects.
+-->
+
+- [Stellar Rails](https://github.com/cryptomover-software/stellar-rails-wallet) - an open source web browser based Stellar wallet
+
```

Comment:
Hi @koinbros-filecoin,

Do you have any examples of projects you've built that are live that we can see?

Reply:
yep, the wallet we have built is still being hosted on www.cryptomover.tech

Reply:
Hi @gavinito, 
Your proposal budget is really large and your team doesn’t have any evidence of d3 or libp2p knowledge for the network QoS related items. Not sure a Stellar Rails wallet is enough previous work. This is not likely to get funded. Perhaps you could propose a project with a much smaller budget to start, such as in the $5k range as a proof of concept.